frontend/src/components/NavBar.js

```diff
@@ -204,14 +204,16 @@ class NavBar extends React.Component<{
 
     handleSearchFocus = () => {
         // Disable the terminal to avoid the search bar text being mirrored in the terminal
-        if (
-            $('#command_line')
-                .terminal()
-                .enabled()
-        ) {
-            $('#command_line')
-                .terminal()
-                .focus(false);
+        if ($('#command_line').length > 0) {
```

Comment:
Let's add `const $cmd = $('#command_line');` at the beginning and rewrite the code in terms of `$cmd` to make this code cleaner.